<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data</class>
 <widget class="QDialog" name="Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(134,237,231);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="communicationButton">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777214</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: rgb(134,237,231); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:focus { background-color: rgb(244,236,2); }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pingbao.qrc">
          <normaloff>:/new/prefix1/ICON/6/1 (16).png</normaloff>:/new/prefix1/ICON/6/1 (16).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;通信测试&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="historyButton">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777214</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: rgb(134,237,231); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:focus { background-color: rgb(244,236,2); }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pingbao.qrc">
          <normaloff>:/new/prefix1/ICON/6/1 (50).png</normaloff>:/new/prefix1/ICON/6/1 (50).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;历史数据&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="dataTransmassionButton">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777214</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: rgb(134,237,231); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:focus { background-color: rgb(244,236,2); }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pingbao.qrc">
          <normaloff>:/new/prefix1/ICON/6/1 (51).png</normaloff>:/new/prefix1/ICON/6/1 (51).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;数据发送&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="dataClearButton">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777214</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: rgb(134,237,231); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:focus { background-color: rgb(244,236,2); }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pingbao.qrc">
          <normaloff>:/new/prefix1/ICON/6/1 (58).png</normaloff>:/new/prefix1/ICON/6/1 (58).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;数据清除&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="dataStatixticsButton">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777214</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: rgb(134,237,231); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:focus { background-color: rgb(244,236,2); }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pingbao.qrc">
          <normaloff>:/new/prefix1/ICON/6/1 (24).png</normaloff>:/new/prefix1/ICON/6/1 (24).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;数据统计&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pingbao.qrc"/>
 </resources>
 <connections/>
</ui>
